### PR TITLE
Infinispan README updates

### PR DESCRIPTION
**Pull request by gunnarmorling**
**Check list**:

Your pull request:

- [x] targets the `development` branch
- [x] uses the `999-SNAPSHOT` version of Quarkus
- [x] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [x] has native tests (`mvn clean verify -Pnative`)
- [x] makes sure the associated guide must not be updated
- [x] links the guide update pull request (if needed)
- [x] updates or creates the `README.md` file (with build and run instructions)
- [x] for new quickstart, is located in the directory _component-quickstart_
- [x] for new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

**Comment by gunnarmorling** [comment]
// CC @gsmet @geoand

**Review by geoand — APPROVED** [review]

File: infinispan-client-quickstart/README.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ You have two options:
 **Option 1:** Running with Docker `docker run -it -p 11222:11222 -e USER="Titus Bramble" -e PASS="Shambles" infinispan/server:latest`
 
 There is a known issue between Docker For Mac and Infinispan Client integration. Explanations can be found in
-the following blog post [here](https://blog.infinispan.org/2018/03/accessing-infinispan-inside-docker-for.html)
+[this blog post](https://blog.infinispan.org/2018/03/accessing-infinispan-inside-docker-for.html).
 You **won't need to do this in production**, but for Docker for Mac users we have to configure the following 
-property: `quarkus.infinispan-client.client-intelligence=BASIC`
+property in the `file src/main/resources/META-INF/resources/hotrod-client.properties`: `quarkus.infinispan-client.client-intelligence=BASIC`
```
Comment (gsmet) [inline infinispan-client-quickstart/README.md:14]:
```suggestion
property in the file `src/main/resources/META-INF/resources/hotrod-client.properties`: `quarkus.infinispan-client.client-intelligence=BASIC`
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Spotted a minor formatting issue. Could you take care of it? Also there's a typo in the commit message.

**Comment by gunnarmorling** [comment]
Thx, good spot, added your change. Can you just apply the commit message change by squashing both commits?

**Review by gsmet — APPROVED** [review]

**Pull request merged**